docs/how-tos/manage-fleet.md

```diff
@@ -25,69 +25,156 @@ For example, you might have 30 robots in one warehouse and 500 in another.
+1. [Manage access with role-based access control](#manage-access-with-role-based-access-control)
+1. [Monitoring and debugging](/how-tos/manage-fleet/#monitoring-and-debugging)
+
+{{% /alert %}}
+
+## Prerequisites
+
+{{% expand "A Viam account" %}}
 
 Go to the [Viam app](https://app.viam.com) and sign up with Google, GitHub, Apple, or an email address.
 
-{{% /tablestep %}}
-{{% tablestep link="/cloud/" %}}
-**2. Create organizations and locations**
+{{% /expand%}}
+
+## Organize your machines
+
+Before you start connecting your devices to the Viam app, you need to decide how you want to group your devices.
 
-Use [organizations](/cloud/organizations/), and [locations](/cloud/locations/) within them, to organize your machines into groups and manage user access.
+In the Viam app, {{< glossary_tooltip term_id="machine" text="machines" >}} are grouped into _locations_, and locations are grouped into _organizations_:
+
+- Each location can represent either a physical location or some other conceptual grouping.
+- An organization is the highest level grouping, and often contains all the locations (and machines) of an entire company.
+
+These groupings allow you to manage permissions; you can grant a user access to an individual machine, to all the machines in a location, or to everything in an entire organization.
+You choose how to group your machines.
 
 {{<imgproc src="/fleet/fleet.svg" class="fill aligncenter" resize="800x" style="max-width: 600px" declaredimensions=true alt="Two locations within an organization">}}
 
+If you'd like to look at an example, see [Monitor Air Quality with a Fleet of Sensors](/tutorials/control/air-quality-fleet/#example).
+
+{{< table >}}
+{{% tablestep link="/cloud/organizations/" %}}
+**1. Create organizations**
+
+1. Log into [Viam app](https://app.viam.com) in a web browser.
+1. Click the dropdown in the upper-right corner of the **FLEET** page and use the **+** button to create a new organization.
+   Name the organization and click **Create**.
+1. Create additional organizations as needed.
+
 {{% /tablestep %}}
+{{% tablestep link="/cloud/locations/" %}}
+**2. Create locations**
+
+1. Click **FLEET** in the upper-left corner of the page.
+   A new location called `First Location` is automatically generated for you.
+
+   Click the pencil icon next to the location name and give it a name for what it represents for your use case.
+   Then click **Save**.
+
+1. Create additional locations as needed using **New location** interface, on the left of the **FLEET** page.
+
+{{% /tablestep %}}
+{{% tablestep link="/cloud/locations/" %}}
+**3. Create sub-locations**
+
+If needed, you can add further sub-locations to, for example, differentiate groups of machines within an office.
+
+To add a sub-location:
+
+1. Add a new location using the same **New location** interface.
+
+1. At the bottom of the location’s page, use the New Parent Location dropdown to choose a parent location.
```

Comment:
```suggestion
1. At the bottom of the location’s page, use the **New parent location** dropdown to choose a parent location.
```
[nit] Since it's sentence case in the app